src/lib/structures/CommandStore.ts

```diff
@@ -47,6 +49,31 @@ export class CommandStore extends AliasStore<Command> {
 			}
 		}
 
+		// Reset the registry's contents
+		piece.applicationCommandRegistry.chatInputCommands.clear();
+		piece.applicationCommandRegistry.contextMenuCommands.clear();
+		piece.applicationCommandRegistry['apiCalls'].length = 0;
```

Comment:
Why is bracket notation being used here (amongst other places)? 🤔

Reply:
Private property access without TS yelling that it's private

Reply:
Oh, didn't know about that one. Thanks for sharing 👍